documentation/3.3/content/userguide/aks/_index.md

```diff
@@ -270,3 +289,27 @@ In the **Review + create blade**, review the details you provided for deploying
 If you want to use this template to automate the deployment, download it by selecting **Download a template for automation**.
 
 Click **Create** to create this offer. This process may take 30 to 60 minutes.
+
+
+#### Useful resources
```

Comment:
You need some text between these two headings. I can be something simple like, "Review the following useful resources."

Reply:
See https://github.com/oracle/weblogic-kubernetes-operator/pull/3174/commits/83051c4dc5feb07e74d5e8ac0e3ddcd05d60b3ec